R/method-ggplot-add.R

```diff
@@ -18,8 +18,12 @@ ggplot_add.ggbreak_params <- function(object, plot, object_name) {
     return(plot)
 }
 
-##' @method ggplot_add ggbreak
-##' @export
-ggplot_add.ggbreak <- function(object, plot, object_name) {
-    ggplot_add(print(object), print(plot), object_name)
-}
+## ##' @method ggplot_add ggbreak
+## ##' @export
+## ggplot_add.ggbreak <- function(object, plot, object_name) {
+##    if (inherits(plot, "ggbreak")){
+##        ggplot_add(print(object), print(plot), object_name)
+##   }else{
+##        ggplot_add(print(object), plot, object_name)
+##    }
+##}
```

Comment:
reason to comment out?

this method is to compatible with patchwork

Reply:
Because it will conflict with `plot_list`, The codes will fall into an endless loop.